README.CN.md

```
@@ -1,74 +1,125 @@
 
-<hr />
+<a href="https://state-in-url.dev" target="_blank">演示页面</a> |
+<a href="https://state-in-url.netlify.app/" target="_blank">镜像链接</a>
 
-添加⭐️来支持项目！
+<a href="https://github.com/asmyshlyaev177/state-in-url/blob/master/Limits.md"  target="_blank">URI 长度限制： <b>安全上限为 12KB</b> </a>
 
 <hr />
 
-</div>
+点个 <a href="#"><img src="https://raw.githubusercontent.com/acervenky/animated-github-badges/master/assets/starbadge.gif" width="25" height="25"></a>  并且 <a href="https://github.com/asmyshlyaev177" target="_blank">关注我</a> 来支持这个项目!
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)